samples/bluetooth/fast_pair/locator_tag/Kconfig.sysbuild

```diff
@@ -10,6 +10,29 @@ config BT_FAST_PAIR
 config APP_DFU
 	bool "Enable MCUmgr OTA DFU over Bluetooth"
```

Comment:
The `SUIT` Kconfig option does not exist in the sysbuild Kconfig space

Reply:
@tomchy, could we add the SUIT Kconfig option to the sysbuild Kconfig space? (top-level option defined in the following file: https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.suit)

Reply:
I think it would be great to have the sysbuild SUIT option before this commit goes in. If it's not possible, then maybe we could temporarily use e.g. `SB_CONFIG_SUIT_ENVELOPE` and switch to `SB_CONFIG_SUIT` in another PR (to avoid reworking sample's DFU code later)